.gitignore

```diff
@@ -178,4 +178,4 @@ pip-log.txt
 config.lua
 
 # We don't want to merge any data changes
-**/data
+#**/data
```

Comment:
this change is redundant, remove that

Reply:
Are you sure?
Because when it was enable, I wasn't able to upload the special food files.

Reply:
makes sense, I probably added that so changes to datapack arent considered by git, in this case remove all of that, even the comment

Reply:
About the foods, yes, you missed, they are already added

https://github.com/nekiro/forgottenserver/blob/e54ce9d255e9c7fc558a61bb268fce214962c284/data/actions/actions.xml#L86-L95